### PR TITLE
21948 - Change Continuation Application

### PR DESCRIPTION
**Pull request by ArwenQin**
*Issue #:* /bcgov/entity#21948

*Description of changes:*
- Changed "Continuation In" to "Continuation Application" in create UI

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by ArwenQin** [comment]
Create UI after change:
image

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-721-t3j7ql6o.web.app

**Comment by severinbeauvais** [comment]
Please rebase so that you pick up Karim's latest config change, and check whether the testing-coverage error is now fixed.

**Review by severinbeauvais — APPROVED** [review]
Looks good. Don't merge until we get Andy's thoughts on the NR request type label.

File: src/components/common/NameRequestInfo.vue [hunk 1]
```diff
@@ -306,7 +306,7 @@ export default class NameRequestInfo extends Mixins(CommonMixin, DateMixin) {
       case NrRequestActionCodes.NEW_BUSINESS: return 'New Business'
       case NrRequestActionCodes.RESTORE: return 'Restoration Request'
       case NrRequestActionCodes.AMALGAMATE: return 'Amalgamation'
-      case NrRequestActionCodes.MOVE: return 'Continuation In'
+      case NrRequestActionCodes.MOVE: return 'Continuation Application'
```
[the remark this answers is not in the document]
Comment (ArwenQin) [reply src/components/common/NameRequestInfo.vue:309]: Confirmed with Andy, we will leave this one as 'Continuation In'

**Comment by ArwenQin** [comment]
> Looks good. Don't merge until we get Andy's thoughts on the NR request type label.

Thanks you Severin. I just confirmed with Andy and changed the name request code back.
Rebased.
Could you please double check if we can merge now? Thank you.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-721-t3j7ql6o.web.app

**Pull request merged**